src/core/execute.c

```diff
@@ -2925,6 +2942,9 @@ void exec_context_init(ExecContext *c) {
         c->umask = 0022;
         c->ioprio = IOPRIO_PRIO_VALUE(IOPRIO_CLASS_BE, 0);
         c->cpu_sched_policy = SCHED_OTHER;
+        c->std_input_fdname = NULL;
+        c->std_output_fdname = NULL;
+        c->std_error_fdname = NULL;
```

Comment:
No need to do that, everything is 0 initally. We only initalize stuff that is non-zero.